services/workflows-service/scripts/workflows/parent-kyb-workflow.ts

```diff
@@ -78,7 +78,7 @@ export const kybParentDynamicExample = {
                 countryOfIncorporation: entity.data.countryOfIncorporation,
                 companyNumber: entity.data.registrationNumber,
                 state: entity.data.additionalInfo.company.state
-                vendor: 'open-corporates'
+                vendor: 'veriff'
```

Comment:
Why? It's a request to OpenCorporates, not to Veriff

Reply:
@MatanYadaev  Post merge,havent touched it